### PR TITLE
{perf}[gompi/2021a] Score-P v7.0

### PR DESCRIPTION
**Pull request by robert-mijakovic**
(created using `eb --new-pr`)
Depends on:
- [x] #13425
- [x] #13427
- [x] #13429

### CODE REVIEW

**Comment by maxim-masterov** [comment]
@boegelbot please test @ generoso

**Comment by SebastianAchilles** [comment]
@boegelbot please test @ generoso

**Comment by maxim-masterov** [comment]
If I'm not mistaken, a lot of easyconfigs use `libunwind/1.4.0-GCCcore-10.3.0` in 2021a, shouldn't this easyconfig use v1.4.0 instead of v1.5.0 too?

File: easybuild/easyconfigs/s/Score-P/Score-P-7.0-gompi-2021a.eb [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+toolchain = {'name': 'gompi', 'version': '2021a'}
+
+source_urls = ['https://perftools.pages.jsc.fz-juelich.de/cicd/scorep/tags/scorep-%(version)s']
+sources = ['scorep-%(version)s.tar.gz']
+checksums = ['68f24a68eb6f94eaecf500e17448f566031946deab74f2cba072ee8368af0996']
+
+dependencies = [
+    ('CubeLib', '4.6'),
+    ('CubeWriter', '4.6'),
+    ('libunwind', '1.5.0'),
```
Comment (SebastianAchilles) [inline easybuild/easyconfigs/s/Score-P/Score-P-7.0-gompi-2021a.eb:34]:
```suggestion
    ('libunwind', '1.4.0'),
```

**Review by SebastianAchilles — COMMENTED** [review]
Let's stick to the existing `libunwind/1.4.0`.

**Comment by satishskamath** [comment]
@robert-mijakovic and @SebastianAchilles  any updates on this PR yet?

**Comment by SebastianAchilles** [comment]
@boegelbot please test @ generoso

**Review by SebastianAchilles — APPROVED** [review]
lgtm

**Comment by Flamefire** [comment]
Test report by @Flamefire
**SUCCESS**
Build succeeded for 9 out of 9 (1 easyconfigs in total)
taurusi8009 - Linux centos linux 7.9.2009, x86_64, AMD EPYC 7352 24-Core Processor (zen2), Python 2.7.5
See https://gist.github.com/392fcd1885f02c9d791f52b6982fb38d for a full test report.

**Comment by Flamefire** [comment]
Test report by @Flamefire
**SUCCESS**
Build succeeded for 9 out of 9 (1 easyconfigs in total)
taurusi6470.taurus.hrsk.tu-dresden.de - Linux RHEL 7.9, x86_64, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz (haswell), Python 2.7.5
See https://gist.github.com/13746196e379a396d5f3a391b1050e9b for a full test report.

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
jrlogin11.jureca - Linux centos linux 8.3.2011, x86_64, AMD EPYC 7742 64-Core Processor, Python 3.6.8
See https://gist.github.com/6dff4e90839f698e3e29c564eea189de for a full test report.

**Comment by SebastianAchilles** [comment]
Going in, thanks @robert-mijakovic!

**Pull request merged**